packages/account/src/test-utils/launchNode.ts

```diff
@@ -143,44 +143,67 @@ export const launchNode = async ({
 
     const prefix = basePath || os.tmpdir();
     const suffix = basePath ? '' : randomUUID();
-    const tempDirPath = path.join(prefix, '.fuels', suffix);
+    const tempDirPath = path.join(prefix, '.fuels', suffix, 'chainConfigs');
 
     if (chainConfigPath) {
       chainConfigPathToUse = chainConfigPath;
     } else {
       if (!existsSync(tempDirPath)) {
         mkdirSync(tempDirPath, { recursive: true });
       }
-      const tempChainConfigFilePath = path.join(tempDirPath, 'chainConfig.json');
 
-      let chainConfig = defaultChainConfig;
+      let { stateConfigJson } = defaultChainConfigs;
+      const { chainConfigJson, metadataJson } = defaultChainConfigs;
+
+      stateConfigJson = {
+        ...stateConfigJson,
+        coins: [
+          ...stateConfigJson.coins.map((coin) => ({
+            ...coin,
+            amount: '18446744073709551615',
+          })),
+        ],
+        messages: stateConfigJson.messages.map((message) => ({
+          ...message,
+          amount: '18446744073709551615',
+        })),
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any
+      } as any;
 
       // If there's no genesis key, generate one and some coins to the genesis block.
       if (!process.env.GENESIS_SECRET) {
         const pk = Signer.generatePrivateKey();
         const signer = new Signer(pk);
         process.env.GENESIS_SECRET = hexlify(pk);
 
-        chainConfig = {
-          ...defaultChainConfig,
-          initial_state: {
-            ...defaultChainConfig.initial_state,
-            coins: [
-              ...defaultChainConfig.initial_state.coins,
-              {
-                owner: signer.address.toHexString(),
-                amount: toHex(1_000_000_000),
-                asset_id: BaseAssetId,
-              },
-            ],
-          },
-        };
+        stateConfigJson.coins.push({
+          tx_id: hexlify(randomBytes(34)),
+          owner: signer.address.toHexString(),
+          // eslint-disable-next-line @typescript-eslint/no-explicit-any
+          amount: '18446744073709551615' as any,
```

Comment:
Because now the state chain config JSON file accepts only numbers when defining UTXOs and message amounts.

This hack was required because all amounts need to be defined as U64 and in the number format. 

However JS does not support U64, and when the JSON file is read, JS messes up all almont. 

So I am defining the amounts as strings to avoid JS mutating the values. Later, after stringifying everything I use a regex to remove `'` from the string numbers.

I am using `any` so I can force all amounts to be a string since TS is not happy because, in the JSON file, they are numbers